WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -514,40 +513,107 @@ - (void)refreshPullToRefresh
     }
 }
 
+#pragma mark - No Results Views
+
+- (void)initNoResultsViews
+{
+    self.noResultsViewController = [NoResultsViewController controller];
+    self.noConnectionViewController = [NoResultsViewController controller];
+}
+
 - (void)refreshNoResultsView
 {
-    [self.noResultsViewController removeFromView];
-    
     if (![self contentIsEmpty]) {
+        [self.tableView setHidden:FALSE];
```

Comment:
Seems like our convention is to use `YES` and `NO` vs `TRUE` and `FALSE`.  An ack shows very few uses of `FALSE` in our code (more in third party pods) but lots and lots of uses of `NO`. 
I guess strictly speaking YES and NO are BOOL types and TRUE and FALSE are Boolean types.  NSHipster has a page about it. https://nshipster.com/bool/  🤷

Reply:
> Seems like our convention is to use YES and NO vs TRUE and FALSE.

Ugh, I always get these backward. Sorry.